Examples/test-suite/errors/pp_deprecated.i

```diff
@@ -1,9 +0,0 @@
-%module xxx
-
-
-%extern ext;
-
-#warning Print this warning
-
-#error This is an error
```

Comment:
This seemed to be the only test of #error, so should be added in somewhere else if the entire testcase is removed.

Reply:
Good spot, I'll add one.

(I did investigate and conclude that there wasn't an alternative deprecated feature which could be used for the case this is nominally testing - it needs one for which the warning is emitted at preprocessing time, and none of the remaining ones are.)

Reply:
Done: 365a540fdbfe741ca50066f68b24c67ee7f56322

I've included coverage we didn't have before for `#error` and `#warning` in the inactive branch of `#if` (like we already had for unknown preprocessor directives).